### PR TITLE
Add 8.0.1 release notes for Fleet and Elastic Agent

### PR DESCRIPTION
**Pull request by dedemorton**
Adds release notes for 8.0.1.

### CODE REVIEW

File: docs/en/ingest-management/release-notes/release-notes-8.0.asciidoc [hunk 1]
```diff
@@ -12,14 +12,41 @@
...
+
+{fleet}::
+* Avoid breaking the {fleet} setup when a compatible package is unavailable in
+the registry, but the cluster already has a version of the package installed.
+{kib-pull}125525[#125525]
+* Tell users when assets are installed in a different space. {kib-pull}125278[#125278]
+
+{agent}::
+* Fix artifact verification failure that prevented corrupted artifacts from
+being removed and downloaded again. {agent-pull}30281[#30281]
```
Comment (ph) [inline docs/en/ingest-management/release-notes/release-notes-8.0.asciidoc:44]: This is not part of 8.0.1

**Review by ph — CHANGES_REQUESTED** [review]

**Review by jen-huang — APPROVED** [review]
Fleet LGTM

File: docs/en/ingest-management/release-notes/release-notes-8.0.asciidoc (hunk 1 above)
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.0.asciidoc:44]:
```suggestion
```

File: docs/en/ingest-management/release-notes/release-notes-8.0.asciidoc [hunk 2]
```diff
@@ -12,14 +12,37 @@
...
+{fleet}::
+* Avoid breaking the {fleet} setup when a compatible package is unavailable in
+the registry, but the cluster already has a version of the package installed.
+{kib-pull}125525[#125525]
+* Tell users when assets are installed in a different space. {kib-pull}125278[#125278]
+
+// end 8.0.1 relnotes
+
+
+// begin 8.0.1 relnotes
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.0.asciidoc:45]:
```suggestion
// begin 8.0.0 relnotes
```

**Review by ph — APPROVED** [review]

**Comment by ph** [comment]
LGTM thanks @dedemorton

**Pull request merged**